test/cpp/jit/test_misc.h

```diff
@@ -1121,8 +1121,8 @@ void testGraphExecutor() {
   ASSERT_EQ(stack.size(), 2);
   at::Tensor r0, r1;
   std::tie(r0, r1) = lstm(input, hx, cx, w_ih, w_hh);
-  ASSERT_TRUE(almostEqual(Variable(stack[0].toTensor()).data(), r0));
-  ASSERT_TRUE(almostEqual(Variable(stack[1].toTensor()).data(), r1));
+  ASSERT_TRUE(almostEqual(stack[0].toTensor(), v(r0)));
+  ASSERT_TRUE(almostEqual(stack[1].toTensor(), v(r1)));
```

Comment:
`almostEqual()` performs subtraction between the two tensors, which requires the two tensors to have the same type. Since `stack[0].toTensor()` and `stack[1].toTensor()` are Variables, we need to also convert `r0` and `r1` to Variables to perform the comparison.